src/Creator/RefundCreator.php

```diff
@@ -22,14 +22,14 @@
 use Sylius\RefundPlugin\Provider\RemainingTotalProviderInterface;
 use Webmozart\Assert\Assert;
 
-final class RefundCreator implements RefundCreatorInterface
+final readonly class RefundCreator implements RefundCreatorInterface
 {
     /** @param OrderRepositoryInterface<OrderInterface> $orderRepository */
     public function __construct(
-        private readonly RefundFactoryInterface $refundFactory,
-        private readonly RemainingTotalProviderInterface $remainingTotalProvider,
-        private readonly OrderRepositoryInterface $orderRepository,
-        private readonly ObjectManager $refundManager,
+        private RefundFactoryInterface $refundFactory,
+        private RemainingTotalProviderInterface $remainingTotalProvider,
+        private OrderRepositoryInterface $orderRepository,
+        private ObjectManager $refundManager,
```

Comment:
I think it makes sense to replace every `ObjectManager` with `EntityManagerInterface` since odm support is dropped. (There are 12 instances of `ObjectManager` in the codebase.)

```suggestion
        private EntityManagerInterface $refundManager,
```

Reply:
@diimpp done in #452